x-pack/plugins/infra/public/containers/logging_legacy/state/entries/actions.ts

```diff
@@ -6,12 +6,18 @@
 
```

Comment:
Could `logging_legacy` be moved to `logging` at this point?

Reply:
I would like to skimp on that here because we are about to move these file anyway when moving the redux store to the top level.

Reply:
Agreed, makes sense.